dasharo-security/tpm2-commands.robot

```diff
@@ -13,6 +13,7 @@ Resource            ../rtectrl-rest-api/rtectrl.robot
 Resource            ../variables.robot
 Resource            ../keywords.robot
 Resource            ../keys.robot
+Resource	    ../lib/bios/menus.robot
```

Comment:
@DaniilKl I do not think that this is necessary, we already add this robot file in keywords.robot.

Reply:
Deleted here 2dd9c61d16532b6f907307ce2293dde54dffec91